validation/validate_map.py

```diff
@@ -0,0 +1,202 @@
+'''
+Description: Validate the BPC to cBioPortal mapping file. 
+Author: Haley Hunter-Zinck
+Date: 2022-01-27
+'''
+
+import argparse
+import logging
+import pandas as pd
+import yaml
+import logging
+import re
+
+import synapseclient
+from synapseclient import Synapse
+from synapseclient.core.exceptions import (
+    SynapseAuthenticationError,
+    SynapseNoCredentialsError,
+)
```

Comment:
Linting thing: pep8 standards state imports go in order of: separated by spaces.
* System imports
* installed packages
* local modules


```suggestion
import argparse
import logging
import re
import yaml

import pandas as pd
import synapseclient
from synapseclient import Synapse
from synapseclient.core.exceptions import (
    SynapseAuthenticationError,
    SynapseNoCredentialsError,
)
```